migrations/1528395642_add_user_id_to_campaign_plan.up.sql

```diff
@@ -1,8 +1,22 @@
 BEGIN;
 
+ALTER TABLE campaign_plans ADD COLUMN user_id integer;
+
+-- Set to first site admin if not attached
+UPDATE campaign_plans SET user_id = users.id
```

Comment:
What does this do if you have multiple site admins? What if the number of campaign plans is higher than site admins? [Read the "Notes" here](https://www.postgresql.org/docs/9.6/sql-update.html#notes)

Doesn't the `WHERE` in this query apply to `campaign_plans`?

I thought you could do something like this:

```suggestion
UPDATE campaign_plans SET user_id = (SELECT id FROM users where site_admin LIMIT 1) WHERE NOT EXISTS (SELECT 1 FROM campaigns WHERE campaigns.campaign_plan_id = campaign_plans.id);
```